prep.sh

```diff
@@ -38,6 +38,20 @@ for LINE in $LINEZ; do
 	cd "megawar-${LINE}"
 	jar c0Mf "../../../target/megawar-${LINE}.war" *
 	popd
+	if [[ ${LINE} != weekly ]]; then
+		PROFILE="-P${LINE}"
+	fi
+	# TODO https://github.com/jenkinsci/maven-hpi-plugin/pull/464
```

Comment:
This is a prototype, so it is probably OK to consume an incremental build here. But if others feel strongly otherwise, I can release a new version of Maven HPI plugin, update the plugin parent POM to consume it, and remove the `-Dhpi-plugin.version` hack from this file. I have a feeling this code might change again soon, so this seems easier for now, but it is harmless to have this `-DcommitHashes` option in Maven HPI plugin since this is the only consumer.

Reply:
I think this is just fine for this stage of the prototype.